Loop/View Controllers/StatusTableViewController.swift

```diff
@@ -211,9 +211,11 @@ class StatusTableViewController: UITableViewController, UIGestureRecognizerDeleg
                 resVol = reservoirVolume {
                 reservoirLevel = min(1, max(0, Double(resVol / Double(capacity))))
             }
-            
-            if let status = dataManager.latestPumpStatus {
-                reservoirLevel = Double(status.reservoirRemainingPercent) / 100
+
+            reservoirLevel = dataManager.latestReservoirValue?.unitVolume
```

Comment:
And it's mixing whats intended to be a percent with a unit volume. (I can't wait for the iOS 10 `Measurement` type)